### DISCUSSION TITLE
Code snippet support?

### DISCUSSION THREAD

**Discussion by dok9612** [typst/typst#3916] [Q&A]
I was wondering if typst also offers some features where I could create notes of both mathematics (LaTeX) and maybe a code snippet like in jupyter notebook where I could 'run' codes and also be able to see codes written in clean, formatted snippets as notetaking. Is this something that is available?

**Comment by Enter-tainer** [comment]
Can you explain what you are looking for?

**Reply by Enivex** [reply]
It sounds like they're looking for Quarto to be honest.
